tests/counter/CounterCommandModel.ts

```diff
@@ -1,22 +1,18 @@
-// @ts-nocheck FIXME
-// https://github.com/dubzzz/fast-check/issues/2781
-import fc from "https://cdn.skypack.dev/fast-check@3.12.0";
-
-import { Chain, types } from "https://deno.land/x/clarinet@v1.7.1/index.ts";
+import { Simnet } from "@hirosystems/clarinet-sdk";
+import { Cl, PrincipalCV, UIntCV } from "@stacks/transactions";
+import fc from "fast-check";
 
 export type Stub = {
   counter: number;
 };
 
 export type Real = {
-  chain: Chain;
+  simnet: Simnet;
 };
 
 export type CounterCommand = fc.Command<Stub, Real>;
 
-//
 // Interop: Clarity <-> TypeScript
-//
 
 export class Uint {

```

Comment:
On a future PR, I'll get rid of these types and just use `Cl` directly. I think there's no benefit, after all, having those wrapper types. On sBTC I'm just using [`types` (the old equivalent of `Cl`) directly](https://github.com/stacks-network/sbtc/pull/152/files#diff-ad5217b1068abdeaa8ae7e66d0291ed368b68076faf612e23c5c84c448f6da41R66) and worked out pretty well.